ext/intl/uchar/uchar.stub.php

```diff
@@ -35,11 +35,17 @@ public static function chr(int|string $codepoint) {}
     /** @return int|false|null */
     public static function digit(int|string $codepoint, int $radix = 10) {}
 
-    /** @return bool|null */
-    public static function enumCharNames(int|string $start, int|string $limit, callable $callback, int $nameChoice = IntlChar::UNICODE_CHAR_NAME) {}
+    /**
+     * @param callable $callback
+     * @return bool|null
+     */
+    public static function enumCharNames(int|string $start, int|string $limit, $callback, int $nameChoice = IntlChar::UNICODE_CHAR_NAME) {}
```

Comment:
Hopefully I remember well that we can't use `callable` as a type for method parameters

Reply:
We should not add new uses of callable in methods until it's fixed, but there's no reason to remove existing ones :)

Reply:
Or maybe there is ... probably we should just fix the bug ^^